app/src/main/java/com/kelvin/bdaypost/util/SharedPrefsUtil.kt

```diff
@@ -0,0 +1,22 @@
+package com.kelvin.bdaypost.util
+
+import android.content.Context
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class SharedPrefsUtil @Inject constructor(@ApplicationContext private val context: Context) {
+
+    fun putString(key: String, value: String) {
+        context
+            .getSharedPreferences(SHARED_PREF_KEY, Context.MODE_PRIVATE)
+            .edit()
+            .putString(key, value)
+            .apply()
+    }
+
+    companion object {
+        const val SHARED_PREF_KEY = "bday_post_shared_pref"
```

Comment:
can we label this var something like `BDAY_PREF_KEY" just in case we add more keys later its not so generic. If you wanted to, you can even lift the function of the getSharedPref out so later you can swap them but i think that's over kill for now, this is fine till we need to use multiple files later

Reply:
Yeah makes sense, updated the name